src/robusta/integrations/kubernetes/custom_models.py

```diff
@@ -166,23 +166,23 @@ def exec(self, shell_command: str, container: str = None) -> str:
 
         return exec_shell_command(self.metadata.name, shell_command, self.metadata.namespace, container)
 
-    def get_logs(
-        self,
-        container=None,
-        previous=None,
-        tail_lines=None,
-        regex_replacer_patterns: Optional[List[NamedRegexPattern]] = None,
-        regex_replacement_style: Optional[RegexReplacementStyle] = None,
-        filter_regex: Optional[str] = None,
+    @staticmethod
+    def find_logs(
+            name: str,
+            namespace: str,
+            container=None,
```

Comment:
what happens to the function when the container is None?

Reply:
if the function fails I might add 
```
if container is None:
 raise exception...
```

Reply:
I have checked the necessity for a valid container name and it is not mandatory. It didnt throw any exceptions.
But in the file block we add the log name like this: `FileBlock(f"{pod.metadata.name}/{container}.log"` for which i have added some conditional file naming logic